Capri.Web/Controllers/PromoterController.cs

```diff
@@ -68,6 +73,11 @@ public IActionResult GetAll()
             Guid id,
             [FromBody] PromoterRegistration newData)
         {
+            if(newData == null || !ModelState.IsValid)
+            {
+                return BadRequest("You provided invalid data");
```

Comment:
Same as here, impersonal form will be better.

Reply:
Done